pkg/controller/logstorage/logstorage_controller.go

```diff
@@ -617,6 +651,50 @@ func (r *ReconcileLogStorage) deleteInvalidECKManagedPublicCertSecret(ctx contex
 	return r.client.Delete(ctx, secret)
 }
 
+func (r *ReconcileLogStorage) getElasticsearchCertificateSecrets(ctx context.Context) (*corev1.Secret, *corev1.Secret, *corev1.Secret, error) {
```

Comment:
Can you add a function comment to list the secrets and order that are returned?

Reply:
Sure thing